Zend/Optimizer/zend_dump.c

```diff
@@ -732,8 +740,10 @@ static void zend_dump_op_line(const zend_op_array *op_array, const zend_basic_bl
 		}
 	}
 
-	zend_dump_op(op_array, b, opline, dump_flags, ssa, ssa_op);
-	fprintf(stderr, "\n");
+	zend_dump_op(op_array, b, opline, dump_flags &~ ZEND_DUMP_LINE_NUMBERS, ssa, ssa_op);
```

Comment:
Shouldn't be needed anymore.

Reply:
Fixed in 6a3898195f29dd11b98f3a3f46d10c5ec732f881

Reply:
Sorry, actually fixed in efa2568042807eb811bab1c607daaefd77bffb24

Only printing line numbers in op_line function, which is what you meant ...